### PR TITLE
Ocrvs 2160

### PR DESCRIPTION
**Pull request by mushrafulhoque-dsi**

### CODE REVIEW

File: packages/client/src/views/RegisterForm/RegisterForm.tsx [hunk 1]
```diff
@@ -267,25 +267,28 @@ class RegisterFormView extends React.Component<FullProps, State> {
...
+          )
         }
-        return { ...memo, [field.name]: fieldTouched }
-      },
-      {}
-    )
+      }
+      return { ...memo, [field.name]: fieldTouched }
+    }, {})
+    console.log(touched)
```
Comment (youshamahmood96) [inline packages/client/src/views/RegisterForm/RegisterForm.tsx:291]: log needs to be removed

File: packages/client/src/views/RegisterForm/RegisterForm.tsx [hunk 2]
```diff
@@ -445,6 +448,7 @@ class RegisterFormView extends React.Component<FullProps, State> {
         groupHasError = hasFormError(activeSectionFields, activeSectionValues)
       }
       if (groupHasError) {
+        console.log(groupHasError)
```
Comment (youshamahmood96) [inline packages/client/src/views/RegisterForm/RegisterForm.tsx:451]: log needs to be removed

**Review by youshamahmood96 — CHANGES_REQUESTED** [review]

**Review by youshamahmood96 — APPROVED** [review]

File: packages/client/src/views/RegisterForm/RegisterForm.tsx (hunk 1 above)
Comment (mushrafulhoque-dsi) [reply packages/client/src/views/RegisterForm/RegisterForm.tsx:291]: Thanks. log removed

File: packages/client/src/pdfRenderer/transformer/offlineTransformer.ts [hunk 3]
```diff
@@ -127,6 +128,7 @@ export const offlineTransformers: IFunctionTransformer = {
     payload?: TransformerPayload,
     optionalData?: IAvailableCountries[]
   ) => {
+    console.log(templateData)
```
Comment (Kayumuzzaman) [inline packages/client/src/pdfRenderer/transformer/offlineTransformer.ts:131]: log should be removed.

**Review by Kayumuzzaman — CHANGES_REQUESTED** [review]

**Review by Kayumuzzaman — APPROVED** [review]

**Pull request merged**